python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
+
+def _get(self, key):
+    """
+    If the key is a string, search the Schema for the correct key object and return the 
+    column view
+    """
+    if isinstance(key, basestring):
+        return self[self.schema.find(key).key]
+    return self[key]
+
+def _setitem_(self, key, value):
+    """
+    Set the values of a column
+    """
+    self.get(key)[:] = value
+
+def _set(self, key, value):
+    """
+    Set the value of a column
+    """
+    self[key] = value
+
+def _extract(self, *patterns, **kwds):
+    """
+    Extract a dictionary of {<name>: <column-array>} in which the field names
+    match the given shell-style glob pattern(s).
```

Comment:
I'm very grateful for the docs. However, as a minor style point: the first sentence is a brief description (and usually starts right after the initial `"""` rather than on its own line). Then a blank line, then details (if any).